db/schema.rb

```diff
@@ -77,6 +77,7 @@
     t.integer "created_by_user_id"
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
+    t.json "dublin_core"
```

Comment:
I wonder if this should have gone into the `Dataset` table rather than `Work`...or perhaps we don't even need the `Dataset` table.

Reply:
The `Dataset` table has a field to indicate the profile (e.g. DublinCore vs Data Cite) and I am wondering if we should move that field to the `Work` table.

Reply:
See ERD: https://docs.google.com/drawings/d/1q2sfj8rrcNVgqQPK5uT_t79A9SYqncinh3HbnCSGMyQ/edit